From e5f1a9972a8e22d7c2d5da160d5193711ea2ba2b Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Thu, 13 Nov 2014 01:53:43 -0500
Subject: [md] Revert: dm-cache: add call to mark_tech_preview

Message-id: <1415843623-26132-10-git-send-email-snitzer@redhat.com>
Patchwork-id: 99889
O-Subject: [RHEL7.1 PATCH 9/9] dm cache: Revert "[md] dm-cache: add call to mark_tech_preview"
Bugzilla: 1159001
RH-Acked-by: Joe Thornber <thornber@redhat.com>

BZ: 1159001

This reverts rhel7.git commit 27e42e507a6bf49503817031a6d6c35416c07fca.

As of RHEL7.1 the DM cache target is fully supported.

Signed-off-by: Mike Snitzer <snitzer@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/md/dm-cache-target.c b/drivers/md/dm-cache-target.c
index 8369b73..ea6c35a 100644
--- a/drivers/md/dm-cache-target.c
+++ b/drivers/md/dm-cache-target.c
@@ -2535,7 +2535,6 @@ static int cache_ctr(struct dm_target *ti, unsigned argc, char **argv)
  int r = -EINVAL;
  struct cache_args *ca;
  struct cache *cache = NULL;
- static bool seen = false;
 
  ca = kzalloc(sizeof(*ca), GFP_KERNEL);
  if (!ca) {
@@ -2560,10 +2559,6 @@ static int cache_ctr(struct dm_target *ti, unsigned argc, char **argv)
 
  ti->private = cache;
 
- if (!seen) {
-  mark_tech_preview("DM cache", THIS_MODULE);
-  seen = true;
- }
 out:
  destroy_cache_args(ca);
  return r;
-- 
1.7.1